README.md

```diff
@@ -1,11 +1,84 @@
 ## What?
+
 This is the future place for Katuma web interface. It will be a universal web app make in node and ReactJS
 
-=======
 This project is based on [React Redux universal hot example](https://github.com/erikras/react-redux-universal-hot-example).
 
+---
+
 ## DEBUG Node
+
 Put `debugger` in the server code you want to debug and then run this on a Terminal.
+
 ```
 ./node_modules/node-inspector/bin/node-debug.js --debug-port=5999
 ```
+
+---
+
+## Development on Mac OS X
+
+### Via Vagrant
+
+    $ cd ~/Sites/coopdevs/katuma-provisioning
+    $ vagrant up
+
+#### Backend (Ruby on Rails)
+
+    $ vagrant ssh
+    $ cd katuma/
+    $ bundle exec rails s --binding 0.0.0.0
+    $ localhost:3000
+
+#### Frontend (Node JS)
+
+    $ vagrant ssh
+    $ cd katuma-web/
+    $ npm run dev
+    $ localhost:8000
+
+---
+
+### Via Local "a pelo"
```

Comment:
let's talk in a month, when you'll be scared to delete `node_modules` directory.... :trollface: